src/Makefile

```diff
@@ -73,7 +73,7 @@ clean Clean:
 
 
 Pure: Clean
-	$(RM) h/auto.h
+	$(RM) h/auto.h h/revision.h
```

Comment:
This will break out-of-tree builds. The only reason revision.h was introduced is to preserve revision info for when you create a source distribution (`make dist`) where you don't have git data anymore

Reply:
i.e, you don't want someone building from source distribution ending up with no revision.h accidentally when the do `make clean`

Reply:
OK, I'll take it out.